templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -237,7 +237,7 @@ if [[ "$TEST" = "azure" ]]; then
   {%- if pulp_scheme == "https" %}
   AZCERTIFI=$(/opt/az/bin/python3 -c 'import certifi; print(certifi.where())')
```

Comment:
Gerrod and I had a call to deconstruct some of this code, we had a difficult time figuring out exactly what was going on but here's the current best idea:

* https://pypi.org/project/certifi/ is self-contained and doesn't allow the extension or modification of the trust store

* therefore this code appears to operate on the somewhat hacky principle of replacing certifi's trust store behind it's back by appending or rewriting the certificate file.  See the hacks on line 222-224

* it's doing that for both the "azcertifi" certificate on the CI host, and the certifi certificate on the CI image

Reply:
I tried this method for finding the correct certifi python path for the container. https://github.com/pulp/pulp_python/pull/645/files#diff-02059c243ab1ae5faa6a95ec965f0b351e0f6c96e5452c90881b6b7e3ee11618R169

Reply:
Thanks, I was trying to avoid the round trips into the container but I don't think it's going to be simpler than just doing this.